src/graphics/render_stl.cpp

```diff
@@ -594,7 +600,7 @@ int write_scene_stl(Stl_context& stl_context, cmzn_scene_id scene,
 	else
 	{
 		display_message(ERROR_MESSAGE,"write_scene_stl.  Missing scene");
-		return_code = 0;
+        return_code = CMZN_ERROR_GENERAL;

```

Comment:
Consider making this a method on the Stl_context; even though many other functions it calls are not.
If using public error codes here, need to check the other paths; for_each_graphics_object_in_scene_tree produces 0/1, so need to ensure return_code is set to either 1 -> CMZN_OK or 0 -> CMZN_ERROR_GENERAL.
Also update docs to say:
```
@return Result OK on success, ERROR_ARGUMENT if invalid scene, otherwise ERROR_GENERAL if failed.
```
Please also fix typo in name of Graphicis_object_to_stl that this method calls.